itest/tests/exportResults.test.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import fsExtra from "fs-extra"
+
+import dialog from "spectron-fake-dialog"
+import path from "path"
+
+import {
+  click,
+  newAppInstance,
+  pcapIngestSample,
+  startApp,
+  startSearch,
+  writeSearch
+} from "../lib/app"
+import {locator} from "../../src/js/test/integration"
+import {retryUntil} from "../lib/control"
+
+describe("test exporting results", () => {
+  let app
+  let testIdx = 0
+
+  beforeAll(async () => {
+    app = newAppInstance(path.basename(__filename), ++testIdx)
+    dialog.apply(app)
+    await startApp(app)
+    await pcapIngestSample(app)
```

Comment:
FYI I'm trying to get rid of `pcapIngestSample()` in favor of `ingestFile()` everywhere as part of #796. If you could use `ingestFile()` here that would be helpful.

Reply:
Now that #796 is in, if you rebase on master, you'll need to use `ingestFile()`.